_data/sites.json

```diff
@@ -1691,7 +1691,7 @@
         "name": "Blogger",
         "url": "https://support.google.com/blogger/answer/41932",
         "difficulty": "impossible",
-        "notes": "You can't delete your Blogger Account without deleting your entire Google Account. But you can delete your blog.",
+        "notes": "You can't delete your Blogger Account without deleting your entire Google Account. But you can delete your blog.\nIf you want to delete your entire google account, use the instructions for <a  href=\"https://justdeleteme.xyz/#google\" target=\"_self\">Google</a>.",
```

Comment:
Why the `target=self`, so it opens on the same page?

Reply:
Yeah, not really necessary, but that's what it does

Reply:
done @tupaschoal, anything else I should change?